configuration/boot_folder.md

```diff
@@ -0,0 +1,36 @@
+# The Boot folder
+
+In a basic Raspbian install the boot files are stored on the first partition of the SD card, which is VFAT formated. This means that it can be read on both Windows and Linux devices. 
+
+When the Raspberry Pi is powered on, it loads various files from the boot partition/folder in order to start up the various processors, then boots the Linux kernel.
+
+## Boot Folder contents
+
+### bootcode.bin
+
+The bootloader. Loaded by the SoC on boot, does some very basic setup, and subsequently loads one of the start*.elf files.
+
+### start.elf, start_x.elf, start_db.elf, start_cd.elf
+
+These are binary blobs (firmware) that are loaded on to the VideoCore in the SoC, which then take over the boot process.
+start.elf is the basic firmware, start_x.elf includes camera drivers and codec, start_db.elf is a debug version of the firmware, and start_cd.elf is a cutdown version, with no support hardware blocks like codecs and 3d, and for use when gpu_mem=16 is specified in config.txt. More information on how to use these can be found in [config.txt](./config-txt/boot.md).
```

Comment:
"drivers and codec" -> "drivers and codecs" ?
"no support hardware blocks" -> "no support for hardware blocks" ?
3d -> 3D ?

Is it worth adding backticks around the filenames to give them `code` formatting?